lib/1.4/utility.dml

```diff
@@ -301,20 +335,19 @@ for power-on reset, hard reset and soft reset, respectively.
 
 #### Related Templates
 
-power\_on\_reset, hard\_reset, soft\_reset
+state\_signal, power\_on\_reset, hard\_reset, soft\_reset
 
 </section>
 */
 
 // Instantiate this on top level to support soft reset.
 template sreset is soft_reset {
-    port SRESET {
+    port SRESET is state_signal {
         implement signal {
             method signal_raise() {
```

Comment:
```suggestion
            method signal_raise() default {
```

Reply:
Agreed. Fixed and updated documentation of hreset, sreset and poreset templates.